### PR TITLE
docs: Showcase auto-generated ids

### PR DESCRIPTION
**Pull request by JigarJoshi**

### CODE REVIEW

File: docs/quickstart/with-java.mdx [hunk 1]
```diff
@@ -120,33 +123,37 @@ public class TigrisApp {
     TigrisCollection<User> users = helloDB.getCollection(User.class);
 
     // insert
-    users.insert(new User(1, "Jania McGrory", 6045.7));
-    users.insert(new User(2, "Bunny Instone", 2948.87));
+    InsertResponse<User> insertJaniaResponse = users.insert(new User( "Jania McGrory", 6045.7));
+    InsertResponse<User> insertBunnyResponse = users.insert(new User("Bunny Instone", 2948.87));
+
+    // read the auto-generated ids
+    int janiaId = insertJaniaResponse.getDocs().get(0).getId();
```
Comment (ovaistariq) [inline docs/quickstart/with-java.mdx:130]: Can we have a better way of fetching the ID from the response? I expect most of the users to use auto-generated IDs. In that case it is best to have a nicer way of fetching the ID?

Comment (JigarJoshi) [reply docs/quickstart/with-java.mdx]: we discussed this and updated the client to do inplace update. updated docs accordingly.

File: docs/quickstart/with-java.mdx [hunk 2]
```diff
@@ -97,124 +98,121 @@ import java.util.Objects;
...
+    public static void main(String[] args) throws TigrisException {
+        // configuration
+        TigrisConfiguration config =
+                TigrisConfiguration.newBuilder("localhost:8081")
+                        .withNetwork(
+                                TigrisConfiguration.NetworkConfig.newBuilder()
+                                        .usePlainText() // for dev env - plaintext communication
+                                        .build())
+                        .build();
+        // construct client
```
Comment (ovaistariq) [inline docs/quickstart/with-java.mdx:110]: Could you add back the newlines, it is easier to read that way.

File: docs/quickstart/with-java.mdx [hunk 3]
```diff
@@ -97,124 +98,121 @@ import java.util.Objects;
...
+        helloDB.createOrUpdateCollections(User.class);
+        // get collection
+        TigrisCollection<User> users = helloDB.getCollection(User.class);
+        // insert
+        User user1 = new User("Jania McGrory", 6045.7);
+        User user2 = new User("Bunny Instone", 2948.87);
+        InsertResponse<User> insertJaniaResponse = users.insert(user1);
+        InsertResponse<User> insertBunnyResponse = users.insert(user2);
+        // read the auto-generated ids
+        int janiaId = user1.getId();
```
Comment (ovaistariq) [inline docs/quickstart/with-java.mdx:124]: Do we need these variables defined or can we directly use the getId() function call wherever this is needed

Comment (JigarJoshi) [reply docs/quickstart/with-java.mdx:124]: This is just to show-case that the IDs are generated and filled in for you. We can ofcourse do it directly on user when required.

File: docs/quickstart/with-java.mdx (hunk 2 above)
Comment (JigarJoshi) [reply docs/quickstart/with-java.mdx:112]: IDE copy paste lost formatting. fixed it.

File: docs/quickstart/with-java.mdx [hunk 4]
```diff
@@ -97,124 +98,132 @@ import java.util.Objects;
...
+                                TigrisConfiguration.NetworkConfig.newBuilder()
+                                        .usePlainText() // for dev env - plaintext communication
+                                        .build())
+                        .build();
+
+        // construct client
+        TigrisClient client = StandardTigrisClient.getInstance(config);
+
+        // create or get db
+        TigrisDatabase helloDB = client.createDatabaseIfNotExists("asdasdsad");
```
Comment (ovaistariq) [inline docs/quickstart/with-java.mdx:116]: Let's choose some other name

Comment (JigarJoshi) [reply docs/quickstart/with-java.mdx]: ofcourse. thanks for catching this. (This was to trigger new db creation)

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**